config/config.yaml

```diff
@@ -10,7 +10,8 @@ project_settings:
   backup_project_id: '<your-project-id>'
   restoration_project_id: '<your-project-id>'
   authorized_requestor_service_accounts: ['sa-eligible-to-read-data@your-project-id.iam.gserviceaccount.com']
+  debug_mode: True
```

Comment:
By default I would change debug_mode as False

Reply:
In fact you should use prod configuration as the default one

Reply:
Done, both :)